app/scripts/inpage.js

```diff
@@ -56,6 +67,14 @@ if (shouldInjectProvider()) {
     target: CONTENT_SCRIPT,
   });
 
+  // this is currently equivalent to process.env.MULTICHAIN_API
+  ///: BEGIN:ONLY_INCLUDE_IF(build-beta,build-flask)
+  getMultichainClient({
+    transport: getDefaultTransport(),
+  }).then((client) => {
+    registerSolanaWalletStandard({ client });
+  });
+  ///: END:ONLY_INCLUDE_IF
```

Comment:
nit. can we move to after the EVM initializeProvider call in case getMultichainClient throws? that way at least window.ethereum isn't also broken?

Reply:
or I guess try catch haha

Reply:
https://github.com/MetaMask/metamask-extension/pull/31705/commits/4f0e9db0383d080f51a766ffeda8934e01613c62